util/deprecated/v1beta1/patch/doc.go

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
Note:

Had to delete the following unit test files:
- util/deprecated/v1beta1/patch/patch_test.go
- util/deprecated/v1beta1/patch/suite_test.go

Due to incompatibility with our internal envtest package which assumes some scheme versions and not being trivial to workaround..

Reply:
Resolved